docs/programmers-guide.rst

```diff
@@ -68,7 +74,7 @@ ThreeFry
 Sobol
     The (scrambled) 32- and 64-bit sobol quasi-random number generators generated the result from :math:`d` dimensions by flattening them into the output. The result at offset :math:`n` in memory is generated from offset :math:`n\;\mathrm{mod}\; d` in dimension :math:`\lfloor n / d \rfloor`, where :math:`d` is the generator's number of dimensions.
 
-cuRAND Compatibility
+cuRAND compatibility
```

Comment:
cuRAND compatiblity should be in reference section

Reply:
spoke to Stanley about it and he insists that cuRAND compatibility must be a part of rocRAND documentation because
rocRAND technically supports being run on CUDA, so hypothetically it could be easy for a CUDA user to compare the two and rocRAND is still a direct competitor to cuRAND.

Having said that he thinks cuRAND compatibility should be a part of hipRAND document as well.

Reply:
Placed cuRAND compatibility under reference.